tests/settings.py

```diff
@@ -94,6 +94,11 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+# middleware list for newer djangos
+MIDDLEWARE = list(MIDDLEWARE_CLASSES)
```

Comment:
It's already a list so why the need of `list()`?

Reply:
It's here to make a copy of the list and not get bitten by `.remove()` side effects later.